ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift

```diff
@@ -411,24 +430,35 @@ open class InputViewController: UIInputViewController, KeymanWebDelegate {
   func showHelpBubble(afterDelay delay: TimeInterval) {
     keymanWeb.showHelpBubble(afterDelay: delay)
   }
-  
-//  func setCursorRange(_ range: NSRange) {
-//    keymanWeb.setCursorRange(range)
-//  }
-  
-  func setText(_ text: String?) {
-    keymanWeb.setText(text)
-  }
-  
+
   func clearText() {
-    setText(nil)
-    setSelectionRange(NSRange(location: 0, length: 0), manually: true)
+    setContextState(text: nil, range: NSRange(location: 0, length: 0))
     log.info("Cleared text.")
```

Comment:
How come you don't call `keymanWeb.resetContext()` like you do in #1756?
```
func clearText() {
  setContextState(text: nil, range: NSRange(location: 0, length: 0))
  keymanWeb.resetContext()
  log.info("Cleared text.")
}
```

Reply:
That one's a result of merging in #1764, which needed to be based on this PR's original (alpha-targeted) version.  If we want, I could probably duplicate that work here with no ill effect, but it's a fix for a predictive text issue.